.github/dockerfiles/ov_test/ubuntu_24_04_x64/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM openvinogithubactions.azurecr.io/dockerhub/ubuntu:24.04
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+RUN apt-get update && \
+    apt-get install software-properties-common && \
+    apt-get update && \
+    apt-get install \
+        curl \
+        git \
+        gpg-agent \
+        tzdata \
+        # Python
+        python3-full \
+        libhdf5-dev \
+        && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install build dependencies
+ADD install_build_dependencies.sh /install_build_dependencies.sh
+RUN chmod +x /install_build_dependencies.sh && \
+    /install_build_dependencies.sh && \
```

Comment:
do we need build dependencies for test image? I supposed we need to install install_openvino_dependeices.sh from installation package

Reply:
Good point, changed.

Reply:
is it applicable for other test docker images?

Reply:
Yes, I've updated them as well.